CameraOverlay/OrchestratorClass.py

```diff
@@ -0,0 +1,79 @@
+"Orchestrator Script That Controls The Camera System"
+
+#imports
+import json
+import argparse
+import time
+import paho.mqtt.client as mqtt
+import config
+
+#Get arguments passed into Python script
+
+def get_args(argv=None):
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--host", type=str,
+                        default="192.168.100.100",
+                        help="ip address of the broker")
+    return parser.parse_args(argv)
+
+#Orchestrator Class
+
+class Orchestrator():
+
+    def __init__(self, brokerIP , port = 1883):
+
+        self.BROKER_IP = brokerIP
+        self.port = port
+        self.mqtt_client = None
+
+   
+    def on_connect(self, client, userdata, flags, rc):
+        """The callback for when the client receives a CONNACK response from the server."""
+        print("Connected with result code " + str(rc))
+
+        # Subscribing in on_connect() means that if we lose the connection and
+        # reconnect then subscriptions will be renewed.
+        client.subscribe("camera/set_overlay")
+        client.subscribe("camera/get_overlays")
+
+    def on_message(self, client, userdata, msg):
+        """The callback for when a PUBLISH message is received from the server."""
+        print(msg.topic + " " + str(msg.payload))
+        if msg.topic == "camera/get_overlays":
+            configs = config.read_configs()
+            client.publish("camera/push_overlays", json.dumps(configs))
+        elif msg.topic == "camera/set_overlay":
+            config.set_overlay(json.loads(str(msg.payload.decode("utf-8"))))
+
+    def on_log(self, client, userdata, level, buf):
+        """The callback to log all MQTT information"""
+        print("\nlog: ", buf)
+
+
+    def on_disconnect(self, client, userdata, msg):
+        """ The callback that is called when user is disconnected from broker"""
+        print("Disconnected from broker")
+
+
+    def start(self):
+        "start Orchestrator"
```

Comment:
Use triple quotes here to document a function. Also I'd copy over the empty lines and other comments which were in the original file, as they make the code easier to read/understand.